test/src/test/java/org/corfudb/runtime/view/AbstractViewTest.java

```diff
@@ -350,6 +350,16 @@ public String getEndpoint(int port) {
         return "test:" + port;
     }
 
+    /**
```

Comment:
If you move the logic as suggested in the comment above, this function is no longer necessary.